composer.json

```diff
@@ -41,9 +41,9 @@
         }
     ],
     "require": {
-        "php": "^8.1",
+        "php": ">=8.1",
```

Comment:
It makes no sense to do this within PHP 8, as both will require PHP 8.1.0 or higher.

But the `>=` option may be dangerous in the future, as all versions above 8.1.0 will fall under it. For example, 9, 10, 11 and so on.